### PR TITLE
RIM: Initial rim tool

### PR DESCRIPTION
**Pull request by Nostritius**
This PR adds the rim tool which makes it possible to pack files into rim archives.

### CODE REVIEW

File: man/rim.1 [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.Dd June 27, 2018
```
Comment (DrMcCoy) [inline man/rim.1]: Please use the current date.

**Comment by DrMcCoy** [comment]
Oh, and please also add the rim executable (both extensionless and with .exe) to the .gitignore file.

**Comment by Nostritius** [comment]
The issues should now be fixed

**Comment by Nostritius** [comment]
Ok, now the rim executable is also added to the .gitignore file

**Comment by DrMcCoy** [comment]
Merged as 142c696...9ca0557, thanks! :)

**Pull request closed without merging**